difference_of_squares.rb

```diff
@@ -0,0 +1,34 @@
+class Squares
+  @@num = 0
+
+  private
+
+  def initialize(*num)
+    if num.length != 0
+      @@num = num[0]
+    end
+  end
+
+  def square_of_sum
+    square_sum = 0
+    (1..@@num).each do |i|
+      square_sum += i
+    end
+    return square_sum ** 2
+  end
+
+  def sum_of_squares
+    sum_square = 0
+    (1..@@num).each do |i|
+      sum_square += i ** 2
+    end
+    return sum_square
+  end
+
+  def difference
+    return Squares.new().square_of_sum - Squares.new().sum_of_squares
```

Comment:
sir first i need sum of all numbers till given number so i have use for()
is there any other way???